faker/providers/person/pl_PL/__init__.py

```diff
@@ -774,3 +774,798 @@ def pwz_nurse(self, kind='nurse'):
         kind_char = 'A' if kind == 'midwife' else 'P'
 
         return '{:02d}{}{}'.format(region, ''.join(map(str, core)), kind_char)
+
+    tax_office_codes = (
```

Comment:
I think we can do away with the comments for each tax office code, and then just add the source of these tax office codes in the `nip()` method. That way the list can be compressed to span just a few lines.

Reply:
comments removed, still long list :)

Reply:
It beats having to scroll through an additional 800 lines though.